### PR TITLE
Ingest data from EMQX and HiveMQ brokers and Redpanda

### PR DESCRIPTION
**Pull request by fahadullahshah261**
Hi Docs team! I created integration guides for data ingestion from the EMQX and HiveMQ brokers and Redpanda into RisingWave. I also updated the integration guide of Grafana by adding a section about RW Cloud. Please review them and let me know your feedback and suggestions. Thanks.

### CODE REVIEW

File: docs/ingest/ingest-from-redpanda.md [hunk 1]
```diff
@@ -8,6 +8,127 @@ slug: /ingest-from-redpanda
...
-As Redpanda is fully compatible with Kafka, you can ingest data from Redpanda into RisingWave in the same way as you ingest data from Kafka.
 
-For the syntax, settings, and examples, see [Ingest data from Kafka](ingest-from-kafka.md).
+You can ingest data from [Redpanda](https://redpanda.com), an event streaming platform that supports real-time data streaming. Redpanda Cloud, a fully managed service, provides automated upgrades, patching, data balancing, and built-in connectors. It offers customizable cluster options to meet your data sovereignty, infrastructure, and development needs.
+
+## Setup Redpanda
+
+This guide will walk you through the steps to create a Redpanda cluster on Redpanda Cloud and connect it to RisingWave for data ingestion. For more details about Redpanda, refer to the [Redpanda Documentation](https://docs.redpanda.com/current/get-started/quick-start-cloud/).
+
+### Step 1: Sign Up for Redpanda Cloud
```
Comment (hengm3467) [inline docs/ingest/ingest-from-redpanda.md]: As a general row, we use sentence-style headings in our documentation. This is different from our blog and probably worth noting.

File: docs/guides/grafana-integration.md [hunk 2]
```diff
@@ -63,4 +63,53 @@ GRANT SELECT ON MATERIALIZED VIEW mv_name TO grafanareader;
 
 See the [GRANT](/sql/commands/sql-grant.md) command for more details.
 
-Now that RisingWave is added as a database, you can start creating dashboards within Grafana using the data in RisingWave. For an extensive tutorial that covers how to create dashboards in Grafana with data queried from RisingWave, check out the [Use RisingWave to monitor RisingWave metrics](/tutorials/monitor-rw-metrics.md) tutorial, which uses a demo cluster so you can easily try it out on your device.
\ No newline at end of file
+Now that RisingWave is added as a database, you can start creating dashboards within Grafana using the data in RisingWave. For an extensive tutorial that covers how to create dashboards in Grafana with data queried from RisingWave, check out the [Use RisingWave to monitor RisingWave metrics](/tutorials/monitor-rw-metrics.md) tutorial, which uses a demo cluster so you can easily try it out on your device.
```
Comment (hengm3467) [inline docs/guides/grafana-integration.md]:
```suggestion
Now that RisingWave is added as a database, you can start creating dashboards within Grafana using the data in RisingWave. To learn about how to create dashboards in Grafana with data queried from RisingWave, check out this tutorial: [Use RisingWave to monitor RisingWave metrics](/tutorials/monitor-rw-metrics.md). It details the steps using a demo cluster, which you can try out on your device.
```

File: docs/guides/grafana-integration.md [hunk 3]
```diff
@@ -63,4 +63,53 @@ GRANT SELECT ON MATERIALIZED VIEW mv_name TO grafanareader;
...
+
+## Configure Grafana to read data from RisingWave Cloud
+
+Grafana can also be configured to visualize and monitor data from a RisingWave Cloud cluster, allowing you to build charts and real-time dashboards based on tables and materialized views. Follow these steps to integrate RisingWave as a data source in Grafana.
+
+### Add RisingWave Cloud as a Data Source in Grafana
+
+1. **Access Data Source Settings:**
```
Comment (hengm3467) [inline docs/guides/grafana-integration.md]: These steps should not be bolded. Please reword these to be proper sentences.

Comment (hengm3467) [reply docs/guides/grafana-integration.md]: You can start the step with the sentence, but end it with a period. Basically, stating the purpose and then the actual operations is good, but they shoud be arranged as normal sentences.

**Review by WanYixian — APPROVED** [review]
Adjust sidebar and make sure all images display properly, LGTM now, thanks!

**Pull request merged**